Dangerfile

```diff
@@ -1,37 +1,49 @@
 # frozen_string_literal: true
 
-def release_branch?
-  danger.github.branch_for_base.start_with?('release/') || danger.github.branch_for_base.start_with?('hotfix/')
-end
+github.dismiss_out_of_range_messages
 
-def main_branch?
-  danger.github.branch_for_base == 'trunk'
-end
+# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
+rubocop.lint(files: [], force_exclusion: true, inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
+
+manifest_pr_checker.check_all_manifest_lock_updated
+
+podfile_checker.check_podfile_does_not_have_branch_references
 
-def wip_feature?
-  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
-  has_wip_title = github.pr_title.include?('WIP')
+ios_release_checker.check_core_data_model_changed
```

Comment:
Dangermattic-level note/nitpick. Apologies for not noticing this earlier, but seeing the method in action I do wonder about the naming.

"Check Core Data Model changed" makes me think that we expect the model to have changed. Whereas what we are trying to do is ensure that _if_ the model changed, it happened only before the build has been released for beta testing yet, https://github.com/Automattic/dangermattic/blob/37da73a5d70f2009868a6e4d83d9a4003e9d96c2/lib/dangermattic/plugins/ios_release_checker.rb#L25-L40.

I don't have a strong suggestion for a name that would make this clearer... `alert_on_core_data_model_changed` maybe? I considered `warn_on...` but the type of message is configurable.

Reply:
Or, like `ios_release_checker.check_release_notes_and_app_store_strings` in the line below, a simpler

```ruby
ios_release_checker.check_core_data_model
```

Reply:
Good suggestion -- I've [created an issue](https://github.com/Automattic/dangermattic/issues/56) for it.